vexctl.yaml

```diff
@@ -0,0 +1,32 @@
+package:
+  name: vexctl
+  version: 0.2.3
+  epoch: 0
+  description: A tool to create, transform and attest VEX metadata
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - go
```

Comment:
```suggestion
```
We can remove the runtime dependency on go

Reply:
Will remove. Cooking up a commit! TY